docs/reST/ref/image.rst

```diff
@@ -81,18 +82,19 @@ following formats.
 
    The returned Surface will contain the same color format, colorkey and alpha
    transparency as the file it came from. You will often want to call
-   ``Surface.convert()`` with no arguments, to create a copy that will draw
-   more quickly on the screen.
+   :func:`pygame.Surface.convert()` with no arguments, to create a copy that
+   will draw more quickly on the screen.
 
-   For alpha transparency, like in .png images, use the ``convert_alpha()``
-   method after loading so that the image has per pixel transparency.
+   For alpha transparency, like in .png images, use the
+   :func:`pygame.Surface.convert_alpha()` method after loading so that the
+   image has per pixel transparency.
 
-   pygame may not always be built to support all image formats. At minimum it
-   will support uncompressed ``BMP``. If ``pygame.image.get_extended()``
-   returns 'True', you should be able to load most images (including PNG, JPG
+   Pygame may not always be built to support all image formats. At minimum it
+   will support uncompressed ``BMP``. If :func:`pygame.image.get_extended()`
+   returns ``True``, you should be able to load most images (including PNG, JPG
    and GIF).
 
-   You should use ``os.path.join()`` for compatibility.
+   You should use :func:`os.path.join()` for compatibility.
```

Comment:
Should we recommend pathlib here instead?

`path = Path("data", "images", "player.png")`

Motivations:

 - Pathlib is available since Python 3.4 and we support 3.6+ in pygame.
 - https://treyhunner.com/2018/12/why-you-should-be-using-pathlib/
 - https://towardsdatascience.com/dont-use-python-os-library-any-more-when-pathlib-can-do-141fefb6bdb5
 - https://betterprogramming.pub/should-you-be-using-pathlib-6f3a0fddec7e

I feel like generally the python community is leaning toward pathlib being the better library for path stuff now.

Reply:
I would recommend using neither (for basic usage) If you do “path/to/file.extension” it works everywhere.

I sometimes see pygame beginners where every load image is a bazillion characters long because they do a bunch of path stuff in every single one.

Reply:
I mean really, all files should be in python packages so they can be loaded from memory blobs as well as actual file systems. ..

I'm not bothered if it is left as is, just thought I should mention it since I saw one of these `pathlib` is better than `os.path` articles the other day.

Reply:
Let's keep as it is for now, just because I'm not sure what to replace it with.